EXPLAINER.md

```diff
@@ -220,39 +222,73 @@ function restoreSavedWindows(project, openWindows) {
 
 The most basic question developers may ask to support multi-screen devices is:
 "Does this device have multiple screens that may be used for window placement?"
 The proposed shape for this particularly valuable limited-information query is
-a `Window.isMultiScreen()` method, alongside the existing `screen` attribute.
+a single `Screen.isExtended` boolean, exposed to secure contexts without an
+explicit permission prompt.
 
 ```webidl
-partial interface Window {
-  // NEW: Returns whether the device has multiple connected screens on success. 
-  Promise<boolean> isMultiScreen();  // UAs may prompt for permission.
+partial interface Screen {
+  // NEW: Yields whether the visual workspace extends over 2+ screens.
+  [SecureContext] readonly attribute boolean isExtended;
 };
 ```
 
-This method exposes the minimum information needed to engage multi-screen users,
-and to avoid requesting information and capabilities that are not applicable to
-single-screen users. By returning a promise, user agents can asynchronously
-determine whether sites may access this information, prompt users to decide,
-calculate the resulting value lazily, and reject or resolve accordingly.
+This single bit provides high value to users and developers alike, allowing
+sites to offer multi-screen functionality only when it is applicable, and to
+avoid requesting additional information and capabilities that are not applicable
+to single-screen environments. The value of this information seems to outweigh
+the risks posed by minimally increasing the device fingerprinting surface. See
+more thorough Privacy & Security considerations later in this document.
 
-In the slideshow example, the site may offer specific UI entrypoints for
-single-screen and multi-screen users.
+In the slideshow example, the site may now offer specific UI entrypoints for
+single-screen and multi-screen environments.
 
 ```js
-async function updateSlideshowButtons() {
-  // NEW: Returns whether the device has multiple connected screens on success. 
-  const multiScreenUI = await window.isMultiScreen();  // Show multi-screen UI?
+function updateSlideshowButtons() {
+  // NEW: Yields whether the visual workspace extends over 2+ screens.
+  const multiScreenUI = window.screen.isExtended;  // Offer multi-screen UI?
   document.getElementById("multi-screen-slideshow").hidden = !multiScreenUI;
-  document.getElementById("single-screen-slideshow").hidden = multiScreenUI;    
+  document.getElementById("single-screen-slideshow").hidden = multiScreenUI;
 }
 ```
 
-### Add `getScreens()` to expose information about available screens
+### Add `Screen.onchange`, an event fired when Screen attributes change
+
+Sites must currently poll the existing `Screen` interface for changes, which is
+a development burden. This can easily be solved by adding an event that is fired
+when screen attributes change. The proposed shape is a `Screen.onchange` event,
```

Comment:
The event is called "change" - "Screen.onchange" is not the event itself, but a way to add an event handler